emdump.bat

```diff
@@ -15,6 +18,27 @@
   set EM_PY=python
 )
 
+:: Work around Windows bug https://github.com/microsoft/terminal/issues/15212 : If this
+:: script is invoked via enclosing the invocation in quotes via PATH lookup, then %~f0 and
+:: %~dp0 expansions will not work.
+:: So first try if %~dp0 might work, and if not, manually look up this script from PATH.
+@if exist %~f0 (
+  set MYDIR=%~dp0
+  goto FOUND_MYDIR
+)
+for %%I in (tools\emdump.bat) do (
```

Comment:
This line looks different to the other?  Do you need to run `./tools/create_entry_points.py`?

Reply:
This is due to a construct in `tools\create_entry_points.py` that does

```
# For some tools the entry point doesn't live alongside the python
# script.
entry_remap = {
  'emdump': 'tools/emdump',
  'emprofile': 'tools/emprofile',
  'emdwp': 'tools/emdwp',
  'emnm': 'tools/emnm',
}

...
      if entry_point in entry_remap:
        bat_data = bat_data.replace('%~n0', entry_remap[entry_point].replace('/', '\\'))
```

It is ok even if these look different. We could replace %~n0 in all scripts, not just the ones in tools/ , but that can be a separate improvement if so desired.